open-metadata-implementation/access-services/asset-lineage/asset-lineage-server/src/main/java/org/odpi/openmetadata/accessservices/assetlineage/outtopic/AssetLineagePublisher.java

```diff
@@ -113,7 +113,7 @@ public void publishGlossaryContext(String glossaryTermGUID) throws OCFCheckedExc
         Map<String, Set<GraphContext>> context = glossaryHandler.buildGlossaryTermContext(serverUserName, entityDetail);
 
         if (MapUtils.isEmpty(context)) {
-            log.debug("Context not found for the entity {} ", entityDetail.getGUID());
+            log.error("Context not found for the entity {} ", entityDetail.getGUID());
```

Comment:
Thinking about this better now, does not seem like ERROR. There can be valid entity that does not have relationships and what asset lineage should do is log this as INFO because of the fact that it is filtering it out. Also suggesting to skip sending the lineage event (otherwise we are sending empty lineage events). This also applies for similar method "publishProcessContext"